### ISSUE TITLE
bug: Node showing as online despite never connecting

### ISSUE THREAD

**Issue by Trinnik** [dec0dOS/zero-ui#112]
# Bug Report

**ZeroUI version:**

v1.5.0

latest

**Current behavior:**

If a node is connected to one network and I add that node id to another network without connecting the node, it will display as online with the information from the network it is actually connected to.

![image](https://user-images.githubusercontent.com/6074848/189371531-de680441-1c92-46f9-acf0-842292bea513.png)

**Expected behavior:**

A node that has never connected to a network should not show as online under that network page.

**Steps to reproduce:**

1.) Add a node to a network
2.) Connect the node to that network
3.) Create a new network
4.) Add the node id to that network but do not connect node to network
5.) Node shows online despite never connecting

**Comment by dec0dOS** [comment]
That is not a bug, but a feature. Check out how does the ZeroTier Central works. It is the same. 

Online status means that there is a [VL1 ZeroTier layer connectivity.](https://docs.zerotier.com/zerotier/manual/)

**Comment by Trinnik** [comment]
I apologize if I'm being ignorant but this is not what I see in zerotier central. My zt central looks like this,

![Screenshot_20220910-080933_Chrome~2.png](https://user-images.githubusercontent.com/6074848/189487384-e4232038-8ddd-4e25-8a9e-ed2faec1e4e9.png)

![Screenshot_20220910-080809_Chrome~2.png](https://user-images.githubusercontent.com/6074848/189487394-e2483f91-092b-46d2-b382-578bcc79ce7a.png)

One network shows online, the other shows unknown. Is there something I'm missing?

**Comment by dec0dOS** [comment]
Please reproduce steps above in your post `Steps to reproduce` section on ZeroTier Central. You should get the same result in a while.

**Comment by Trinnik** [comment]
I see that you are correct, however replicating my steps would have left me in circles until chance lined up. When creating networks in ZT central, the controller node that you are added to is random. I had to create networks until two lined up on the same controller. Once they were on the same controller then I was able to replicate this.

I can see now that this is not an issue, simply something that on ZT central only happens by chance. Thank you for you support!
